### PR TITLE
Extension removal

### PR DESCRIPTION
**Pull request by laps15**
Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

File: browse_foreigners.php [hunk 1]
```diff
@@ -68,8 +68,8 @@
     $table,
     $_REQUEST['field'],
     $foreignData,
-    isset($fieldkey) ? $fieldkey : null,
-    isset($data) ? $data : null
+    isset($fieldkey) ? $fieldkey : (string)null,
+    isset($data) ? $data : (string)null
```
Comment (mauriciofauth) [inline browse_foreigners.php]: Wouldn't it be better to use an empty string?

File: libraries/classes/Controllers/Server/ServerDatabasesController.php [hunk 2]
```diff
@@ -110,9 +112,9 @@ public function indexAction()
             'is_create_db_priv' => $GLOBALS['is_create_db_priv'],
             'dbstats' => $this->_dbstats,
             'db_to_create' => $GLOBALS['db_to_create'],
-            'server_collation' => $this->dbi->getServerCollation(),
+            'server_collation' => $GLOBALS['dbi']->getServerCollation(),
             'databases' => isset($databases) ? $databases : null,
-            'dbi' => $this->dbi,
+            'dbi' => $GLOBALS['dbi'],
```
Comment (mauriciofauth) [inline libraries/classes/Controllers/Server/ServerDatabasesController.php]: It seems to me that you are reverting the  dce862d41b3cdbe91afc43ed6b3d0ff8b5127d72 changes. Is this intentional?

Comment (laps15) [reply libraries/classes/Controllers/Server/ServerDatabasesController.php]: No ideia why this hapenned. I'm trying to fix it.

Comment (laps15) [reply libraries/classes/Controllers/Server/ServerDatabasesController.php]: It probably was a rebase with the master branch that placed my changes over yours. Sorry.

**Pull request merged**